### PR TITLE
feat(QTREES-368, QTREES-367 & QTREES-364): Feedback report components

### PR DESCRIPTION
**Pull request by vogelino**
This PR adds the following presentational components:
- **QTREES-368**: Feedback Confirmation Component
- **QTREES-367**: Feedback Modal Component
- **QTREES-364**: Feedback List Item component

### CODE REVIEW

File: src/components/FeedbackReportForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './FeedbackReportForm'
```
Comment (dnsos) [inline src/components/FeedbackReportForm/index.tsx:1]: I've recently noticed that you started using this. Is it simply to have a more descriptive file name than `index.tsx` for the component to work in?

File: src/components/FeedbackReportModal/FeedbackReportModal.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  const { t } = useTranslation('common')
+  const isImageLoadable = useImageLoadsSuccessfully(imageUrl)
+  const showImg = imageUrl && isImageLoadable
+
+  const formattingComponents = {
+    bold: <strong />,
+  }
+
+  return (
+    <Transition appear show={isOpen} as={Fragment}>
```
Comment (dnsos) [inline src/components/FeedbackReportModal/FeedbackReportModal.tsx:37]: That's nice. I'll have to start looking into this transition helper more.

**Review by dnsos — APPROVED** [review]
Great work, looks exactly as it should and the code is very understandable!

File: src/components/FeedbackReportForm/index.tsx (hunk 1 above)
Comment (vogelino) [reply src/components/FeedbackReportForm/index.tsx:1]: Exactly and it makes searching the file easier!

File: src/components/FeedbackReportModal/FeedbackReportModal.tsx (hunk 2 above)
Comment (vogelino) [reply src/components/FeedbackReportModal/FeedbackReportModal.tsx:37]: I think so too! TBH I used this from headless UI examples. It's great!

**Pull request merged**